src/Vies/Contractor/Company/ValueAddedTax/ViesIdentifierFactory.php

```diff
@@ -28,20 +28,20 @@ public function create(string $identifier, string $country): ValueAddedTaxIdenti
     {
         try {
             $validation = $this->vies->validateVat($country, $identifier);
-        } catch (Exception $e) {
-            throw new ViesException('VIES external service exception: ' . $e->getMessage());
-        }
 
-        if ($validation->isValid()) {
+            if (false === $validation->isValid()) {
+                throw new Exception('Invalid Tax Id');
+            }
```

Comment:
Dodałem rzucenie exceptionem kiedy Vat jest niepoprawny aby nie w ogóle nie nastąpiła próba przesłania go do Wfirmy.

Reply:
To tu bym dedykowany exception dał, żeby on nie był łapany zaraz poniżej. Albo po prostu zostawić ten stary tam gdzie był, a tu dodać dedykowany.